client/components/App.jsx

```diff
@@ -1,18 +1,57 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import GoogleMap from './GoogleMap.jsx';
+import SearchBox from './SearchBox.jsx';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
+
+const App = () => {
+  const [state, setState] = useState ({
+    location: {lat: 39, lng: -94},
+    searchBoxText: 'Search a City!'
+  })
```

Comment:
is location referencing current location or the eventual location in the searchBoxText? If it is the former, might want to give searchBoxText its own useState and give location a more descriptive title.

Reply:
based off of the rest of the component looks like it is the former of the 2 scenarios provided.